menas/src/main/scala/za/co/absa/enceladus/menas/controllers/RestExceptionHandler.scala

```diff
@@ -45,4 +49,11 @@ class RestExceptionHandler {
     ResponseEntity.notFound().build[Any]()
   }
 
+  @ExceptionHandler(Array(classOf[OozieActionException]))
+   def handleOozieActionException(ex: RuntimeException): ResponseEntity[Object] = {
+    val err = RestError(ex.getMessage)
+    logger.error(s"Exception: $err", ex.getCause)
```

Comment:
Why log only the cause? Don't you lose some of the stack trace if you only do `ex.getCause`?